verify_stage_sigs

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+. settings
+
+ARCHES="x86_64 source"
+
+EXIT_CODE=0
+
+for os in $OSES; do
+  BASE="tmp/$PROJECT/$VERSION/$os"
```

Comment:
Can we make this a parameter? I don't like that we're spreading `tmp/$PROJECT/$VERSION` all over the code base as hardcoded paths. Looks like `upload_stage_rpms` already has a path `tmp/$PROJECT/$VERSION/` so can we make that `$1`?

Or a new setting?

Reply:
new setting it is!